### PR TITLE
New LitGpt on LLama and Qwen for now

### LINKED ISSUE #14: Support more models
Gemma models require non-standard variants of attention like softmax logit capping. This is currently not supported in yalis.

### PR DESCRIPTION
**Pull request by ishanrev**
Have tested on LLAMA and Qwen I believe it should work for other models as well if any concerns, please let me know I will ensure those models work as well.

### CODE REVIEW

**Comment by prajwal1210** [comment]
Closing as resolved by https://github.com/axonn-ai/yalis/pull/83
